beater/handlers.go

```diff
@@ -316,14 +316,12 @@ func processRequest(r *http.Request, pf ProcessorFactory, prConfig *processor.Co
 		return http.StatusBadRequest, err
 	}
 
-	list, err := processor.Transform(data)
+	payload, err := processor.Decode(data)
 	if err != nil {
 		return http.StatusBadRequest, err
 	}
 
-	if err = report(list); err != nil {
-		return http.StatusServiceUnavailable, err
-	}
+	go report(payload)
```

Comment:
I removed the usage of the channel and instead use a go routine for further processing, as the check to not overfloat the server with requests is now already done at an earlier stage with @roncohen 's changes.

However, as the `transform` and the `publishAll` is done within this go routine, after releasing the request from the channel, and those actions might be more heavy than the `validation` and `decoding` it could lead to more go routines processing than the number of `concurrent_requests` is set to. 
WDYT?